drivers/driver.go

```diff
@@ -17,7 +17,7 @@ type Driver interface {
 	UpdateRecord(database, table, column, value, primaryKeyColumnName, primaryKeyValue string) error
 	DeleteRecord(database, table string, primaryKeyColumnName, primaryKeyValue string) error
 	ExecuteDMLStatement(query string) (string, error)
-	ExecuteQuery(query string) ([][]string, error)
+	ExecuteQuery(query string) ([][]string, int, error)
```

Comment:
I would like to suggest to add named result in return value to make it clearer


```suggestion
	ExecuteQuery(query string) (results [][]string, totalRecords int, err error)
```

Reply:
Well... No other methods in this interface use named return values so it sounds like that suggested improvement is not really related to my changes (and introducing it just for this method would make the code inconsistent which is not something I'd like to do).

So instead, if you think its a valuable addition, I would suggest that you open a PR yourself to update all methods in this interface in one go so everything will remain consistent and the next time someone want to make a change here its clear what format should be used (which is hard for a one-time contributor if the code is inconsistent).

Reply:
I would say I'm agree, but I still disagree.

By adding 1 returned value, things are becoming more complicated. 

A method returning something, plus an error is classic. No need to use named return, everyone understand.

The returned value you add is an interface, but it's unclear what it could be.

I can understand you disagree. It's OK. The same way you disagree with my suggestions.

I don't know if other interfaces are something that I consider unclear.

Now what, I don't think I would be able to use a named return here.

So I have a suggestion. Feel free to apply or ignore.

You could add a comment on the interface to explain the method returns.

If I didn't manage to convince you nevermind... I might open a PR later, or never.

All these was a point of detail.

Reply:
>By adding 1 returned value, things are becoming more complicated.

While that's certainly true, the `GetRecords` method, for example, also has 3 return params. So I'm still not adding a new pattern to this interface in which case I usually try to follow the existing style of the existing code to keep things consistent.

I guess the same goes for adding a comment. While I agree that helps to better understand the methods in this interface, at that same time I think it should then be done consistently to all methods which (in my opinion) is out of scope of this PR as it's a change that is completely not related to what I am trying to fix here.